.github/workflows/build_appliances.yaml

```
@@ -12,9 +12,19 @@ jobs:
       run: |
         git clone https://www.github.com/bdunne/imagefactory.git /build/gha/$GITHUB_RUN_ID/imagefactory
   build_appliances:
+    strategy:
+      max-parallel: 2
+      matrix:
+        target: [azure, ec2, gce, hyperv, libvirt, openstack, ovirt, vagrant, vsphere]
```

while this is acceptable, I think I'd prefer this were listed line-by-line, particularly for future diffs/changes
```suggestion
        target:
        - azure
        - ec2
        - gce
        - hyperv
        - libvirt
        - openstack
        - ovirt
        - vagrant
        - vsphere
```

---

Done